src/SDKs/Azure.ApplicationModel.Configuration/data-plane/Azure.ApplicationModel.Configuration.Tests/samples/Sample7_ConfiguringPipeline.cs

```diff
@@ -52,21 +49,35 @@ public async Task ConfiguringPipeline()
             await client.DeleteAsync("some_key");
         }
 
-        class AddHeaderPolicy : HttpPipelinePolicy
+        class AddHeaderPolicy: HttpPipelinePolicy
         {
-            public override async Task ProcessAsync(HttpMessage message, ReadOnlyMemory<HttpPipelinePolicy> pipeline)
+            private readonly HttpPipelinePolicy _next;
```

Comment:
I really don't think policies should have the next filed. I want policies to be reusable between pipelines (if possible).

Reply:
Why?